SPECS/libbpf/libbpf.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        Libbpf library
 Name:           libbpf
-Version:        0.4.0
-Release:        3%{?dist}
+Version:        1.0.0
+Release:        1%{?dist}
```

Comment:
This is a pretty big version jump, and the release notes indicate that a lot of deprecated APIs have been removed. What's our justification for an upgrade like this that could potentially break things for our customers?

Reply:
Yes, this is a pretty big jump.
The only packages that depend on `libbpf` are `qemu & dwarves`. I've build tested both of them.
IMO, given that `libbpf` is used for `eBPF` most of the tools have been updated and the recommendation is to use the `v1.0` for anything going forward.

Also I had a chat with @chaitanyalala who is looking into eBPF. He suggested that we upgrade to `v1.0`

Reply:
Ha, Chaitanya was actually one of the people I was concerned about breaking, since he's been looking at eBPF recently. Sounds good!